tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/upgrade/animated/AbstractLiferayTableViewCustomPart.java

```diff
@@ -397,7 +397,7 @@ private IPath createPreviewerFile( final String projectName, final IPath srcFile
             for( IFile upgradeFilePath : upgradeFiles )
             {
                 IPath filePath = upgradeFilePath.getLocation();
-                if( isNeedUpgrade( filePath.toFile() ) )
+                if( isNeedUpgrade( filePath.toFile() ) && !filePath.toString().contains( "/target/" ) )
```

Comment:
There could be files in a package like "com.liferay.target.platform" and that would fail this test.  We need to check for the /target/ that is in the sub-dir of the main project folder, not anywhere in the path.